lib/src/api/opt/config.rs

```diff
@@ -0,0 +1,29 @@
+use std::time::Duration;
+
+/// Configuration for server connection, including: strictness, notifications, query_timeout, transaction_timeout
+#[cfg(any(
+	feature = "kv-mem",
+	feature = "kv-tikv",
+	feature = "kv-rocksdb",
+	feature = "kv-speedb",
+	feature = "kv-fdb",
+	feature = "kv-indxdb",
+))]
+#[derive(Debug)]
+pub struct Config {
+	pub(crate) strict: bool,
+	pub(crate) notifications: bool,
+	pub(crate) query_timeout: Option<Duration>,
+	pub(crate) transaction_timeout: Option<Duration>,
+}
```

Comment:
Since these are `pub(crate)` and there's no builder for it, how are these supposed to be constructed?

Reply:
good point! I will make one 👍